tests/realtikvtest/importintotest4/global_sort_test.go

```diff
@@ -46,6 +47,57 @@ func urlEqual(t *testing.T, expected, actual string) {
 	require.Equal(t, urlExpected.String(), urlGot.String())
 }
 
+func assertExternalField(t *testing.T, subtaskMeta any) {
```

Comment:
maybe put into `addindextestutils` pkg, so we can reuse it, and maybe rename the pkg to be `realtikvtestutils`